test/org/centos/contra/Infra/Utils.groovy

```diff
@@ -0,0 +1,369 @@
+import spock.lang.*
+import com.homeaway.devtools.jenkins.testing.JenkinsPipelineSpecification
+
+import org.yaml.snakeyaml.Yaml
+
+public class UtilsSpec extends JenkinsPipelineSpecification {
+    @Shared
+    def config = new HashMap<String, ?>()
+
+    @Shared
+    def infraUtils
+
+    def setup() {
+        infraUtils = loadPipelineScriptForTest("../../src/org/centos/contra/Infra/Utils.groovy")
+        infraUtils.getBinding().setVariable("WORKSPACE", "workspace")
+        infraUtils.getBinding().setVariable("CREDENTIAL_FILE_NAME", "credsfile")
+        infraUtils.getBinding().setVariable("SSH_FILE_NAME", "sshfile")
+        explicitlyMockPipelineStep('container')
+
```

Comment:
Seeing some extra blank lines. A few in the when blocks as well as some in setup blocks